project-base/app/app/preload.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+if (file_exists(dirname(__DIR__) . '/var/cache/prod/App_KernelProdContainer.preload.php')) {
+    require dirname(__DIR__) . '/var/cache/prod/App_KernelProdContainer.preload.php';
```

Comment:
This article (https://symfony.com/blog/php-preloading-and-symfony) says, that you need to add some configuration to php.ini for it to work. Should we add it too?

Reply:
it's already added – https://github.com/shopsys/deployment/blob/main/kubernetes/configmap/production-php-opcache.yaml